src/input.rs

```diff
@@ -1,10 +1,16 @@
-use serde_json::{json, Map};
+use serde::{Deserialize, Serialize};
 use std::{collections::BTreeMap, error::Error, str::FromStr};
 
 use tokei::{Language, LanguageType, Languages};
 
 type LanguageMap = BTreeMap<LanguageType, Language>;
 
+#[derive(Deserialize, Serialize, Debug)]
+struct Output {
+    languages: LanguageMap,
+    totals: Language,
```

Comment:
To remove the breaking change aspect, I wonder if it could use `#[serde(flatten)]` and rename it back to "total" rather than "totals"?

```suggestion
    #[serde(flatten)]
    languages: LanguageMap,
    total: Language,
```

Reply:
That's a good idea! It looks like `flatten` isn't an issue for this. Would you like me to do `#[serde(rename = "Total")]` to preserve the old capitalization for that bit as well?

And this would still be a breaking change since the names for the top level `LanguageType` fields may still be different (`Bash` vs `BASH`), but it keeps the old structure which likely would have been the more obtrusive part of the change

Reply:
Ah true, yeah keeping the same casing makes sense to me.

Reply:
Sounds good. Changes are in 39f40b192337257f48288530d393610cd7232580